### PR TITLE
modify orgFormModal org name regexp

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bug

#### What this PR does / why we need it:
modify the orgFormModal org`s name regexp
the previous regexp don`t match uppercase letter

#### Which issue(s) this PR fixes:

### CODE REVIEW

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (o *OrgFormModal) SetProps() {
 				"rules": []interface{}{
 					map[string]interface{}{
 						"msg":     "由小写字母、数字，-组成(不允许纯数字)",
-						"pattern": "/^(?![0-9]+$)^[^-](.*[^-])?$[a-z0-9-]*$/",
+						"pattern": "/^(?:[a-z]+|[0-9]+[a-z]+|[0-9]+[-]+[a-z0-9])+(?:(?:(?:[-]*)[a-z0-9]+)+)?$/",
```
Comment (sfwn) [inline modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: use const and add unit test for this.

**Review by sfwn — DISMISSED** [review]

**Review by Effet — APPROVED** [review]

File: modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go (hunk 1 above)
Comment (Effet) [reply modules/openapi/component-protocol/scenarios/home-page-sidebar/components/orgFormModal/render.go:105]: this regexp is for js, it's hard to unit test.

and we are planing disable frontend check, all check will in backend.

**Review by sfwn — APPROVED** [review]

**Pull request merged**